### ISSUE TITLE
When running the packaged app that uses SQLite3 in Electron, the following error is prompted.

### ISSUE THREAD

**Issue by cnTrent** [knex/knex#5501]
# Environment

Knex version:2.4.2
Database + version: sqlite3@5.0.11
OS:win

# Bug

 error keno:run rpm install sqlite3 --save 
rip-web package.json is not node-pre-gyp ready" package.json must declare these properties binary

This is my code
function initCache() {
  const knexInstance = knex({
    client: "sqlite3",
    useNullAsDefault: true,
    connection: {
      filename: path.join(app.getPath("userData"), "local.db"),
    },
  });
  /**
   * @Descripttion: 创建一个收获工作台的表
   * @test: test font
   * @msg:
   * @return {*}
   */
  ipcMain.handle("createWorktable", async (_, name) => {
    try {
      await knexInstance.schema.createTable(name, (table) => {
        table.increments("id");
        table.string("box_name");
      });
      return "创建成功";
    } catch (err) {
      return "已经存在收货工作台的表了";
    }
  });
  /**
   * @Descripttion: 往某一张表里插入数据
   * @test: test font
   * @msg:
   * @return {*}
   */

  ipcMain.handle("insertTable", async (_, name, data) => {
    await knexInstance(name).insert({ box_name: data });
  });
  /**
   * @Descripttion: 查找某一个表的所有的数据
   * @test: test font
   * @msg:
   * @return {*}
   */
  ipcMain.handle("allChatSelect", async (_, name, data) => {
    return await knexInstance(name).select("*");
  });
}

When I ran it locally, there was no problem, but when I ran it packaged as an app, there was this error above

**Comment by mercmobily** [comment]
Hi! Thanks for the details and the code sample. This is a duplicate of an existing issue that tracks bundler/Electron guidance; we’ll close this in favour of #6242. Adding a bit of context here that may help you unblock the packaging error.
